README.md

```diff
@@ -256,6 +282,14 @@ o1.micro  |  1  |  1Gi
 o1.nano  |  1  |  512Mi
 o1.small  |  1  |  2Gi
 o1.xlarge  |  4  |  16Gi
+rt1.2xlarge  |  8  |  32Gi
+rt1.4xlarge  |  16  |  64Gi
+rt1.8xlarge  |  32  |  128Gi
+rt1.large  |  2  |  8Gi
+rt1.medium  |  1  |  4Gi
+rt1.micro  |  1  |  1Gi
```

Comment:
shouldn't be here 4Gi memory?

Reply:
the same for small

Reply:
These sizes are intended to be even smaller than `rt1.medium`. Therefore they have a different ratio.